src/lf_mask.rs

```diff
@@ -449,15 +450,17 @@ pub(crate) fn rav1d_create_lf_mask_intra(
     let bx4 = bx & 31;
     let by4 = by & 31;
 
+    let filter_level_yuv = filter_level.0.map(|a| a[0][0]);
+    let [filter_level_y, filter_level_uv] = *<[u16; 2]>::ref_from(&filter_level_yuv).unwrap();
```

Comment:
Why not group these into a single line? I think that would be slightly clearer.

Reply:
Sure, I can do that if you think it's clearer.  I thought it was getting too complex of an expression for a single line.

Reply:
I think having fewer bindings generally makes the code clearer.

Reply:
Hmm, I'm not sure I agree in general.  Variables exist for a reason.

Reply:
But it's the `[u8; 4]` that's the `yuv`, the two Y lumas and two UV chrominances.  `a[0][0]` is only one of them, not all of them.  Then the second line splits them (the `[u8; 4]`) into `y` and `uv` halves and safely transmutes the two `&[u8; 2]`s to `&u16`s.